Doc/library/pathlib.rst

```diff
@@ -96,38 +96,38 @@ Pure path objects provide path-handling operations which don't actually
 access a filesystem.  There are three ways to access these classes, which
 we also call *flavours*:
 
-.. class:: PurePath(*pathsegments)
+.. class:: PurePath(*pathcomponents)
 
    A generic class that represents the system's path flavour (instantiating
    it creates either a :class:`PurePosixPath` or a :class:`PureWindowsPath`)::
 
       >>> PurePath('setup.py')      # Running on a Unix machine
       PurePosixPath('setup.py')
 
-   Each element of *pathsegments* can be either a string representing a
-   path segment, an object implementing the :class:`os.PathLike` interface
+   Each element of *pathcomponents* can be either a string representing a
+   path component, an object implementing the :class:`os.PathLike` interface
    which returns a string, or another path object::
 
       >>> PurePath('foo', 'some/path', 'bar')
       PurePosixPath('foo/some/path/bar')
       >>> PurePath(Path('foo'), Path('bar'))
       PurePosixPath('foo/bar')
 
-   When *pathsegments* is empty, the current directory is assumed::
+   When *pathcomponents* is empty, the current directory is assumed::
 
       >>> PurePath()
       PurePosixPath('.')
 
-   When several absolute paths are given, the last is taken as an anchor
-   (mimicking :func:`os.path.join`'s behaviour)::
+   If a component is an absolute path, all previous components are thrown away
```

Comment:
```suggestion
   If a component is an absolute path, all previous components are ignored
```
"thrown away" feels a bit too informal for the docs

Reply:
I was matching os.path.join https://docs.python.org/3/library/os.path.html#os.path.join, but sure, I can update!